src/components/stepforms/widgets/Autocomplete.vue

```diff
@@ -47,6 +59,9 @@ export default class AutocompleteWidget extends Mixins(FormWidget) {
 
   @Prop({ type: String, default: undefined })
   label!: string;
+
+  @Prop({ type: Boolean, default: false })
+  withExample!: boolean;
```

Comment:
```suggestion
  get  withExample(): boolean {
    return Object.keys(this.options).includes('example')
  };
```

Reply:
Hum, I'm not convinced because I prefer that we keep the option explicit (we may have cases where we want to use the key `example` in the options data structure, but not the templates)